assets/javascripts/discourse/admin/models/ai-persona.js

```diff
@@ -0,0 +1,24 @@
+import RestModel from "discourse/models/rest";
+
+const ATTRIBUTES = [
+  "name",
+  "description",
+  "commands",
+  "system_prompt",
+  "allowed_group_ids",
+  "enabled",
+  "system",
+  "priority",
+];
+
+export default class AiPersona extends RestModel {
```

Comment:
IIRC we are generally trying to move away from `RestModel` because it doesn't support ember `@tracked` properties... I could be wrong though, something to ask David or CvX about

Reply:
it saves so much code though, we need to modernize this thing